src/lib/utils/__tests__/createHelpers-tests.ts

```diff
@@ -34,7 +34,7 @@ describe('insights hogan helper', () => {
     })}>Add to cart</button>
     `;
 
-    expect(output.trim()).toEqual(expected.trim());
+    expect((output as string).trim()).toEqual(expected.trim());
```

Comment:
If TS isn't able to infer the `string` type, we better cast it at the variable declaration line 15.

Reply:
Addressed in f4ca49a9.